docs/specs/offline_store_format.md

```diff
@@ -62,10 +62,10 @@ Here's how Feast types map to BigQuery types when using BigQuery for offline sto
 | Event Timestamp |   `DATETIME` |
 | BYTES | `BYTES` |
 | STRING | `STRING` |
-| INT32 | `INT64` |
+| INT32 | `INTEGER` |
```

Comment:
https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#integer_type INT64 and FLOAT64 seem like correct data type.